pkgs/development/libraries/flatbuffers/generic.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, version
+, sha256
+, patches ? [ ]
+, doCheck ? stdenv.hostPlatform == stdenv.buildPlatform
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flatbuffers";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "flatbuffers";
+    rev = "v${version}";
+    inherit sha256;
+  };
+
+  preConfigure = lib.optional stdenv.buildPlatform.isDarwin ''
```

Comment:
Can we remove the file uncondional?

Reply:
Probably. `BUILD` is almost certainly a bazel file.

Reply:
It seems like this was renamed to `BUILD.bazel`.

Reply:
I removed the `preConfigure` altogether, let's see what happens.

Reply:
We can't remove this, see the top level discussion.